src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -638,7 +630,8 @@ public Object onTraversalSuccess() {
             final long variantContextBufferSize,
             final long segmentSize,
             final long lbSampleIndex,
-            final long ubSampleIndex) {
+            final long ubSampleIndex,
+        final boolean failIfArrayExists) {
```

Comment:
Fix indentation here (there may be literal tabs here that need converting to spaces).